bin/cdk-infra.ts

```diff
@@ -71,7 +71,36 @@ const stack = new FhirWorksStack(app, `smart-fhir-service-${stage}`, {
 });
 // run cdk nag
 Aspects.of(app).add(new AwsSolutionsChecks());
+Aspects.of(app).add(new HIPAASecurityChecks());
 NagSuppressions.addStackSuppressions(stack, [
+    {
+        id: 'HIPAA.Security-IAMNoInlinePolicy',
+        reason: 'We use Inline policies for strict one-to-one relationships between a policy and identity',
+    },
+    {
+        id: 'HIPAA.Security-DynamoDBInBackupPlan',
+        reason: 'Backup is an optional configuration offered alongside the service in backup.ts',
+    },
+    {
+        id: 'HIPAA.Security-LambdaInsideVPC',
+        reason: 'We have a guide for users that would like to deploy resources inside a VPC',
+    },
+    {
+        id: 'HIPAA.Security-OpenSearchInVPCOnly',
+        reason: 'We have a guide for users that would like to deploy resources inside a VPC',
+    },
+    {
+        id: 'HIPAA.Security-S3BucketReplicationEnabled',
+        reason: 'S3 bucket replication is included as best practices in the deployment guide',
+    },
+    {
+        id: 'HIPAA.Security-LambdaConcurrency',
```

Comment:
We should probably create a ticket to track where this lambda is being generated from and ensure it isn't needed in our stack.